posthog/models/remote_config.py

```diff
@@ -279,7 +279,7 @@ def _build_site_apps_js(self):
                 # Indentation to make it more readable (and therefore debuggable)
                 site_functions_js.append(
                     indent_js(
-                        f"\n{{\n  id: '{site_function.id}',\n  init: function(config) {{ return {indent_js(source, indent=4)}().init(config) }} \n}}"
+                        f"\n{{\n  id: '{site_function.id}',\n  type: '{site_function.type}',\n  init: function(config) {{ return {indent_js(source, indent=4)}().init(config) }} \n}}"
```

Comment:
I'd do `json.dumps(site_function.type)` to make sure quotes will always work

Reply:
wrapped both in json.dumps